### PR TITLE
feat(COCO-STQA-68)Added Selection to Hall of Fame

### PR DESCRIPTION
**Pull request by CarbonFanatic**
Added a hardcoded Select in the Hall of fame.
Does not work yet

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/welcome.jsp [hunk 1]
```diff
@@ -23,14 +22,26 @@
...
+        </div>
+        <summary><strong>Hall of Fame</strong></summary>
     <!-- Ryan L. -->
     <table>
         <thead class="thead">
             <tr id="ranking">
-                <th class="text-center">1st</th>
-                <th class="text-center">2nd</th>
-                <th class="text-center">3rd</th>
+                <th id="HOFRank1" class="text-center">1st</th>
```
Comment (Shayminn) [inline src/main/webapp/WEB-INF/jsp/welcome.jsp:42]: Are these IDs necessary for something else?

File: src/main/webapp/WEB-INF/jsp/welcome.jsp [hunk 2]
```diff
@@ -23,14 +22,26 @@
...
+  </span>
+        </div>
+        <summary><strong>Hall of Fame</strong></summary>
     <!-- Ryan L. -->
     <table>
         <thead class="thead">
             <tr id="ranking">
-                <th class="text-center">1st</th>
-                <th class="text-center">2nd</th>
-                <th class="text-center">3rd</th>
```
Comment (Shayminn) [inline src/main/webapp/WEB-INF/jsp/welcome.jsp:32]: Are the following IDs necessary for something else?

**Review by Shayminn — COMMENTED** [review]
Is the functionality working as of right now?
Do you need help with anything? Is anything stopping you from completing this story, if it is still not completed?

**Review by NichitaMantaluta — APPROVED** [review]
Everything looks good to me

**Comment by CarbonFanatic** [comment]
Was forced to Merge with master multiple time so the history is a little all over the place

**Review by Shayminn — APPROVED** [review]
Code reviewed, added some comments, everything else looks good!

**Pull request merged**